slides/02-deploy.qmd

```diff
@@ -41,44 +41,53 @@ knitr:
 ```{r}
 library(tidyverse)
 library(tidymodels)
-library(vetiver)
-library(pins)
-library(countdown)
-
-abalone <- read_csv("abalone.csv")
+library(arrow)
+path <- here::here("data", "inspections.parquet")
+inspections <- read_parquet(path)
 
 set.seed(123)
-ring_split <- initial_split(abalone, prop = 0.8, strata = rings)
-ring_train <- training(ring_split)
-ring_test <- testing(ring_split)
+inspect_split <- initial_split(inspections, prop = 0.8)
+inspect_train <- training(inspect_split)
+inspect_test <- testing(inspect_split)
 
-abalone_fit <-
+inspect_fit <-
   workflow(
-    rings ~ ., 
-    rand_forest(mode = "regression")
+    results ~ facility_type + risk + total_violations, 
+    rand_forest(mode = "classification", trees = 1e3)
   ) |> 
-  fit(data = ring_train) 
+  fit(data = inspect_train)
 ```
 
 ### Python
 
 ```{python}
+## TODO for Isabel: can you add the `inspection_date` predictor to this random forest model?
+## TODO for Isabel: can you add a script to `data-raw` that trains this model and deploys to Connect? see my `prep_chicago_inspections.R` but *only* do model training, not data prep
```

Comment:
Here is TODO 3 for you 3️⃣ 

Let me know if this item doesn't make sense; you can add something like a `prep_chicago_inspections.py` to `data-raw` and keep a script to train a random forest and deploy it to Connect.